rest_framework/pagination.py

```diff
@@ -187,7 +187,9 @@ class PageNumberPagination(BasePagination):
 
     # Set to an integer to limit the maximum page size the client may request.
     # Only relevant if 'page_size_query_param' has also been set.
-    max_page_size = None
+    # Defaults to `None`, meaning page size is unlimited.
+    # It's recommended that you would set a limit to avoid api abuse.
+    max_page_size = api_settings.MAX_PAGE_SIZE
```

Comment:
Same as #8993, I think this should be a property to allow the setting be overriden in Django tests after the views were created.